apps/investigation/kinds/investigation.cue

```diff
@@ -1,96 +1,87 @@
-package kinds
+package investigation
 
 // This is our Investigation definition, which contains metadata about the kind, and the kind's schema
 investigation: {
-	group:      "investigation"
 	kind:       "Investigation"
 	pluralName: "Investigations"
-	scope:      "Namespaced"
-
-	apiResource: {
-		groupOverride: "investigation.grafana.app"
-	}
-
-	codegen: {
-		frontend: false
-		backend:  true
-	}
-
-	current: "v1alpha1"
+	current:    "v1alpha1"
 	versions: {
 		"v1alpha1": {
-			version: "v1alpha1"
-			schema: {
-				// spec is the schema of our resource. The spec should include all the user-ediable information for the kind.
-				spec: #InvestigationSpec
+			codegen: {
```

Comment:
Just a note: you can still set the `codegen` information at the [root level of the kind](https://github.com/grafana/grafana-app-sdk/blob/main/codegen/cuekind/def.cue#L154), it will get applied as the [default value](https://github.com/grafana/grafana-app-sdk/blob/main/codegen/cuekind/def.cue#L132-L137) for each version's `codegen` block.

Reply:
Good point. I'll leave it like this for consistency with playlists, plus I think this is good to keep around to indicate that we support different settings per version (e.g. if devs want to start generating frontend in newer versions).